match/spec/fixtures/existing/certs/distribution/E7P4EE896K.p12

```diff
@@ -1,24 +1,27 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIEogIBAAKCAQEAyTm/v40AZb6I1eXRPVaUF+q683gm+XTRaRC9fjqK3seL117n
```

Comment:
why does it change?

Reply:
This is because we are actually checking the relation of a private key to a public key. This means we need a real key like the one i added.